client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -55,6 +56,7 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 			city: translate( 'City' ),
 			state: translate( 'State / Region / Province' ),
 			zip: translate( 'ZIP code' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Zip Code' )` *ES Score: 15*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jpo-business-address-add-country-field)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).